Backend/AlgorithmsManager.py

```diff
@@ -474,20 +474,42 @@ def get_most_frequent_retweeters(self, username, num_tweets=20, num_retweets=20)
 
         return res
 
+
+
     def get_network_tweets_text(self, id2, username, count, depth):
         textsStr = ""
         if id2 > 0 or username is not None:
             # get bunch of tweets starting with the user of given id
             n = depth  # goes depth level deep
```

Comment:
what is the value of this alias? it seems to me that it makes the code below less clear.